src/packages/@ncigdc/components/RemoveFromCartSingle.js

```diff
@@ -0,0 +1,33 @@
+/* @flow */
+
+import React from 'react';
+import Button from '@ncigdc/uikit/Button';
+import { Tooltip } from '@ncigdc/uikit/Tooltip';
+import { compose } from 'recompose';
+import { connect } from 'react-redux';
+import { toggleFilesInCart } from '@ncigdc/dux/cart';
+
+const styles = {
+  backgroundColor: '#FFF',
+  borderColor: '#CCC',
+  color: '#333',
+  margin: '0 auto',
+  padding: '0px 5px',
+  ':hover': {
+    background:
+      'linear-gradient(to bottom, #ffffff 50%, #e6e6e6 100%) repeat scroll 0 0 #E6E6E6',
+    borderColor: '#ADADAD',
+  },
+};
+
+export default compose(connect(state => state.cart))(({ dispatch, file }) => (
```

Comment:
it's good practive forward props along, like style etc

```
({ dispatch, file, style = {}, ...props }) => ...
```

so users of this component have some control